rat/compiler.py

```diff
@@ -330,14 +330,14 @@ def compile(data_df: pandas.DataFrame, parsed_lines: List[ops.Expr]):
                 elif isinstance(lhs, ops.Param):
                     lower = lhs.lower
                     upper = lhs.upper
-                    assert isinstance(lower, ops.RealConstant)
-                    assert isinstance(upper, ops.RealConstant)
+                    #assert isinstance(lower, ops.RealConstant)
+                    #assert isinstance(upper, ops.RealConstant)
 
                     if lhs_key in variable_indexes:
                         subexpr = variables.Param(lhs_key, variable_indexes[lhs_key])
                     else:
                         subexpr = variables.Param(lhs_key)
-                    subexpr.set_constraints(lower.value, upper.value)
+                    subexpr.set_constraints(float(eval(lower.code())), float(eval(upper.code())))
```

Comment:
@bbbales2 
I've enabled the constraints to be expressions instead of just `RealConstant`. Not a great way to make this work; suggestions welcome.

Reply:
What is this making work? This is like saying it's okay to have expressions as constraints as long as we can evaluate them?

Reply:
Yup, they should be known at compile time. Although right now there are no checks for that.